snmp/assets/monitors/missing_interface_in_octets.json

```diff
@@ -0,0 +1,29 @@
+{
+	"version": 2,
+	"created_at": "2023-12-27",
+	"last_updated_at": "2023-12-27",
+	"title": "Incoming traffic missing data for {{snmp_device.name}} on interface {{interface.name}} in {{device_namespace.name}}",
+	"description": "Alert if missing data for incoming traffic on the network interface",
+	"definition": {
+		"name": "[SNMP] Incoming traffic missing data for {{snmp_device.name}} on interface {{interface.name}} in {{device_namespace.name}}",
+		"type": "query alert",
+		"query": "sum(last_5m):sum:snmp.ifHCInOctets{*} by {device_namespace,snmp_device,interface,interface_alias}.as_count() > 100",
```

Comment:
maybe this isn't has useful as just having the bandwidth usage rate, some of the numbers can have very different ranges (so a single threshold doesn't make sense, given however much more bandwidth each interface has) 

so maybe this isn't as useful as the bandwidth usage rate would also be missing from metrics and notify so this could be redundant 🤔 i was just wondering what would be good / useful even without an explicit threshold to refer back to